src/prpy/util.py

```diff
@@ -2129,3 +2129,75 @@ def GetPointFrom(focus):
         raise ValueError('Focus of the point is an unknown object')
 
     return coord
+
+def ConcatenateTrajectories(robot, traj_list):
+    """
+    Given a list of trajectories for a single manipulator,
+    concatenate them into a single trajectory
+
+    @param traj_list List of consective trajectories for a given manipulator
+    @return base_traj Combined trajectory
+    """
+    from planning.base import Tags
+
+    if len(traj_list) == 0:
+        raise ValueError('Trajectory list is empty')
+
+    env = robot.GetEnv()
+
+    # Create an empty trajectory to populate
+    base_traj = openravepy.RaveCreateTrajectory(env, '')
+    base_cspec = robot.GetActiveConfigurationSpecification('linear')
+    base_traj.Init(base_cspec)
+    #base_cspec = base_traj.GetConfigurationSpecification()
+
+    smoothFlag = True
+    constrainedFlag = False
+    deterministicTrajectoryFlag = True
+    deterministicEndPointFlag = False
+
+    offset = 0
+    for traj in traj_list:
+        cspec_i = traj.GetConfigurationSpecification()
+        if base_cspec != cspec_i:
+            raise ValueError('The configuration specifications of the trajectory do not match')
+
+        tags = GetTrajectoryTags(traj)
+        # Only True if all trajectories have this set
+        if 'smooth' in tags and not tags['smooth']:
+            smoothFlag = False
```

Comment:
This would be skipped if `smooth` is not in `tags`. Is `smooth` by default true?